kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala

```diff
@@ -63,11 +63,10 @@ class ServiceControlCliArguments(args: Seq[String], env: Map[String, String] = s
     // for create action, it only expose Kyuubi service instance to another domain,
     // so we do not use namespace from default conf
     if (action != ServiceControlAction.CREATE && namespace == null) {
-      conf.getOption(HA_ZK_NAMESPACE.key).foreach { v =>
-        if (verbose) {
-          info(s"Zookeeper namespace is not specified, use value from default conf:$v")
-        }
-        namespace = v
+      namespace = conf.get(HA_ZK_NAMESPACE)
+      if (verbose) {
+        info(s"Zookeeper namespace is not specified," +
+          s" use value from default conf:$namespace")
```

Comment:
NOT related to the PR, can we avoid mess up verbose with log4j?

Reply:
> NOT related to the PR, can we avoid mess up verbose with log4j?

IMO, we can have a try, use a custom log level.

Reply:
Yea